applications/wg-easy/Taskfile.yaml

```diff
@@ -221,24 +261,38 @@ tasks:
         done
       
       - echo "Release files prepared in ./release/ directory"
-      
+    deps:
+      - update-version
+
+
   release-create:
     desc: Create and promote a release using the Replicated CLI
     silent: false
-    vars:
-      CHANNEL: '{{.CHANNEL | default "Unstable"}}'
-      RELEASE_NOTES: '{{.RELEASE_NOTES | default "Release created via task release-create"}}'
+    dotenv: ['.env']
     cmds:
-      - echo "Creating and promoting release for {{.APP_NAME}} to channel {{.CHANNEL}}..."
+      - echo "Creating and promoting release for $APP_NAME to channel $CHANNEL..."
       - |
         # Create and promote the release in one step
         echo "Creating release from files in ./release directory..."
-        replicated release create --app {{.APP_NAME}} --yaml-dir ./release --release-notes "{{.RELEASE_NOTES}}" --promote {{.CHANNEL}}
-        
-        echo "Release created and promoted to channel {{.CHANNEL}}"
+        replicated release create --app $APP_NAME --yaml-dir ./release --release-notes "$RELEASE_NOTES" --promote $CHANNEL --version $VERSION
+        echo "Release version $VERSION created and promoted to channel $CHANNEL"
     deps:
       - prepare-release
 
+  release-inspect:
```

Comment:
The docs on our website doesn't really say why this is useful. What do you use this for? Is this used as part of the workflow or is it just wrapping the replicated cli?

Reply:
This is when I create a release and check it in my vendor port, I found a lot of errors from editor lint which may confuse users. That's one the reason to bring release inspect back. If it is not useful, I will remove it

Reply:
I will explain this part in a google doc